locale/en.po

```diff
@@ -2900,7 +2961,8 @@ msgstr "Selected save is upgradeable"
 #: ../src/saving/SaveList.tscn:192
 msgid "SAVE_IS_UPGRADEABLE_DESCRIPTION"
 msgstr ""
-"The selected save is from an older version of Thrive, but can be upgraded.\n"
+"The selected save is from an older version of Thrive, but can be "
+"upgraded.\n"
```

Comment:
This seems like an invalid change, we use 77 columns wide text, and this is not near that, so if you could revert this, that would be nice.

Reply:
The last character is on column 79...

Reply:
I suppose if you count the " and \n part then that is true...